universal-login-web3/test/helpers/createWallet.ts

```diff
@@ -3,6 +3,7 @@ import UniversalLoginSDK, {DeployedWallet} from '@universal-login/sdk';
 import {ETHER_NATIVE_TOKEN, TEST_GAS_PRICE} from '@universal-login/commons';
 
 export const createWallet = async (name: string, sdk: UniversalLoginSDK, wallet: Wallet) => {
+  await sdk.fetchRelayerConfig();
```

Comment:
This shouldn't be here. We definitely should fetch relayer config in the setup phase.